framework/test/unit/specs/application/node/synchronizer/block_synchronization_mechanism/block_synchronization_mechanism.spec.ts

```diff
@@ -1263,6 +1263,22 @@ describe('block_synchronization_mechanism', () => {
 				});
 			});
 		});
+
+		describe('computeBlockHeightsList', () => {
+			it('should return height list for round 0', () => {
```

Comment:
Since this is util function, maybe move the test to utils?
Also, maybe add more detailed expectation as it is pure function